target_score.txt

```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-6
+8
```

Comment:
Yes, that s good switch.

Reply:
> Yes, that s good switch.

No, I do not think so. It is eminent I'd better switch to 4 or 6 later.